### PR TITLE
Fix a compile warning

### PR DESCRIPTION
**Pull request by 23Skidoo**
```
warning: OpentelemetryEcto.setup/1 is undefined (module OpentelemetryEcto is not available or is yet to be defined)
  lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex:22: PrimaOpentelemetryEx.Instrumentation.OpentelemetryEcto.instrument_repo/4
```

### CODE REVIEW

File: lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex [hunk 1]
```diff
@@ -10,15 +10,13 @@ defmodule PrimaOpentelemetryEx.Instrumentation.OpentelemetryEcto do
...
+  	end,
       %{}
     )
   end
-
-  def instrument_repo(_event, _measurements, metadata, _config) do
-    metadata
-    |> Map.fetch!(:opts)
-    |> Keyword.fetch!(:telemetry_prefix)
-    |> OpentelemetryEcto.setup()
```
Comment (cpiemontese) [inline lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex:22]: Maybe this should've been `&__MODULE__` too...? Was that the problem?

File: lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex [hunk 2]
```diff
@@ -10,15 +10,13 @@ defmodule PrimaOpentelemetryEx.Instrumentation.OpentelemetryEcto do
     :telemetry.attach(
       "repo-init-handler",
       [:ecto, :repo, :init],
-      &__MODULE__.instrument_repo/4,
+      fn _event, _measurements, metadata, _config ->
+    	metadata
+    	|> Map.fetch!(:opts)
+    	|> Keyword.fetch!(:telemetry_prefix)
+    	|> OpentelemetryEcto.setup()
+  	end,
```
Comment (cpiemontese) [inline lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex:18]: The formatting seems weird but maybe it's Github

**Review by cpiemontese — COMMENTED** [review]
Thanks for this PR!

Did you manage to test it already?

**Comment by 23Skidoo** [comment]
Yes, I've checked that it compiles and makes the warning go away.

File: lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex (hunk 2 above)
Comment (23Skidoo) [reply lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex]: Should be fixed now.

File: lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex (hunk 1 above)
Comment (23Skidoo) [reply lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex:22]: It's calling a function from the `opentelemetry_ecto` library, not this module. `opentelemetry_ecto` is an optional dependency, so when it's missing, you get a compile warning.

Comment (cpiemontese) [reply lib/prima_opentelemetry_ex/instrumentation/opentelemetry_ecto.ex:22]: ffs :facepalm: I got it, thanks!

**Review by cpiemontese — APPROVED** [review]

**Pull request merged**